src/files/templates/detail.html

```diff
@@ -4,48 +4,76 @@
 
 {% block content %}
 
-  <div class="row mt-3 mb-3 justify-content-start">
-    <div class="col-10">
-      <h3>{{ object.title }}</h3>
-    </div>
-    <div class="col-2 d-flex justify-content-end">
-      <div class="dropdown">
-        <button class="btn btn-outline-primary dropdown-toggle" type="button" id="dropdownActionsButton1" data-bs-toggle="dropdown" aria-expanded="false">
-          Actions
-        </button>
-        <ul class="dropdown-menu" aria-labelledby="dropdownActionsButton1">
-          <li><a class="dropdown-item" href="#">[Add to album]</a></li>
-          <li><a class="dropdown-item" href="{% url 'files:delete' object.uuid %}">Delete</a></li>
-        </ul>
-      </div>
-    </div>
-  </div>
+  <div class="row">
 
-  <div class="row justify-content-between">
     <div class="col-8">
       <div class="row mb-3">
-        <div class="col-11">
-          {% if object.medium_thumbnail.url %}
-            <img src="{{ object.medium_thumbnail.url }}" class="w-100"/>
-          {% else %}
-            <i class="fa-regular fa-eye-slash fa-9x"></i>
-            <p class="lead">No preview yet</p>
-          {% endif %}
+        <div class="col">
+          <h3>{{ object.title }}</h3>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+        {% if object.medium.url %}
+          <img src="{{ object.medium.url }}" class="w-100"/>
+        {% else %}
+          <i class="fa-regular fa-eye-slash fa-9x"></i>
+          <p class="lead">No preview yet</p>
+        {% endif %}
         </div>
       </div>
+    </div>
+
+    <div class="col-4">
       <div class="row mb-3">
-        <div class="col-11">
+        <div class="col d-flex justify-content-end">
+          <div class="dropdown">
+            <button class="btn btn-outline-primary dropdown-toggle" type="button" id="dropdownActionsButton1" data-bs-toggle="dropdown" aria-expanded="false">
+              Actions
+            </button>
+            <ul class="dropdown-menu" aria-labelledby="dropdownActionsButton1">
+              <li><a class="dropdown-item" href="#">[Add to album]</a></li>
+              {% if request.user == object.uploader %}
```

Comment:
all "permission checks" should use object permissions like `publish_basefile` (because it will be possible to grant `publish_basefile` to other users at some point)

Reply:
Done. But it seems there's an issue with permissions. I've "change_file", "delete_file" etc. but the {{ perms }} only show one of them and not all for that object type.